include/wx/msw/bitmap.h

```diff
@@ -177,7 +177,7 @@ class WXDLLIMPEXP_CORE wxBitmap : public wxGDIImage,
     void SetMask(wxMask *mask);
 
     bool HasAlpha() const;
-    void UseAlpha(bool use = true);
+    bool UseAlpha(bool use = true);
     void ResetAlpha() { UseAlpha(false); }
```

Comment:
`override` can be added then!

Reply:
No because `wxBitmap` still doesn't inherit from `wxBitmapBase` in wxMSW, unfortunately.